src/include/OutputFieldsDefault.h

```diff
@@ -258,6 +260,7 @@ class OutputFieldsDefault : public OutputFieldsParams
   int tfield_moments_next_;
   int naccum_ = 0;
   int naccum_moments_ = 0;
+  bool first_time = true;
```

Comment:
The problem with this is that it will now always skip writing the first output, and when a run is started from scratch, that's not what we want. I suppose you might be able to something like
```
if (first_time && grid.timestep() != 0) {
  return;
}
```
and add a comment, too.